app.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START app]
+const express = require('express');
+const fs = require('fs');
+
+var privateKey = fs.readFileSync('key.pem').toString();
+var certificate = fs.readFileSync('cert.pem').toString();
+
+var credentials = {key: privateKey, cert: certificate};
+
+const app = express();
+
+var https = require('https').Server(credentials, app);
+
+
+var io = require('socket.io')(https);
+
+app.use('/src', express.static('src'));
+app.use('/bower_components', express.static(__dirname + '/bower_components'));
+app.use('/node_modules', express.static(__dirname + '/node_modules'));
```

Comment:
That will also serve the node modules with like express etc. perhaps you should should move the bower_components and node_modules along with their bower.json and a new package.json into a sub directory like src/ and serve that?

e.g.
```js
app.use('/bower_components', express.static(__dirname + '/src/bower_components'));
app.use('/node_modules', express.static(__dirname + '/src/node_modules'));
```

and your file structure will look like
```
/
L package.json
  node_modules
  app.js
  src
  L package.json
    bower.json
    node_modules
    bower_components
```

Reply:
I just saw the src app.use. By moving node_modules and bower_components into src, you shouldn't need the other two app.use calls